### PR TITLE
Hot fix - Set Conv Report Vars

### PR DESCRIPTION
**Pull request by mitchute**
Pull request overview
---------------------
 - After converting the surface classification index variables from arrays of ints to arrays of enums, I created `int` report var arrays but forgot to set the values. This corrects that.
 - See #8929 for reference

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/ConvectionCoefficients.cc [hunk 1]
```diff
@@ -4858,6 +4858,9 @@ void ManageInsideAdaptiveConvectionAlgo(EnergyPlusData &state, int const SurfNum
     //  TODO: candidate for rework to do zone level calcs once rather than for each surface
     DynamicIntConvSurfaceClassification(state, SurfNum);
 
+    // Set report var after surface has been classified successfully
+    state.dataSurface->SurfIntConvClassificationRpt(SurfNum) = static_cast<int>(state.dataSurface->SurfIntConvClassification(SurfNum));
+
```
Comment (rraustad) [inline src/EnergyPlus/ConvectionCoefficients.cc:4863]: This looks safe to me. No chance of crash with SurfNum = 0 (would crash all over the place, not just here) and just equating a variable to an int.

**Review by rraustad — APPROVED** [review]
Not sure waiting on CI will show anything other than diffs in these 2 files. I guess it can't hurt to wait, unless merging now will save on the CI backlog.

**Comment by mitchute** [comment]
Agreed. I'm not sure if it will save any CI at this point or not, but I'm going to merge it now regardless.

**Comment by mitchute** [comment]
Thanks for the quick review @rraustad !

**Comment by mjwitte** [comment]
@mitchute This needs another piece to initialize `SurfIntConvClassificationRpt` and `SurfOutConvClassificationRpt` [here](https://github.com/NREL/EnergyPlus/blob/e6cdd73f63d359ddfe52a475bf33c231eb5eccae/src/EnergyPlus/SurfaceGeometry.cc#L1025-L1030). @xuanluo0113 @JasonGlazer This is likely the source of the diffs on your branches.

**Comment by mitchute** [comment]
@mjwitte thanks. I'll push that up right away.

**Comment by rraustad** [comment]
I was also thinking it may be cleaner to put those 2 new lines inside and at the end of each function.

**Comment by mitchute** [comment]
@rraustad are you talking about the ones added yesterday? Here: https://github.com/NREL/EnergyPlus/pull/9035/commits/4f9104c04ae2511683f5627825f627625785ebec

**Comment by rraustad** [comment]
Yes. It's up to you.

**Pull request merged**